platform/representation/src/gestures/DropGesture.js

```diff
@@ -66,8 +69,21 @@ define(
                     );
                 }
             }
+            
+            function shouldCreateVirtualPanel(domainObject){
+                //
+                return domainObject.useCapability('view').filter(function (view){
+                    return view.key==='plot' && domainObject.getModel().type!== 'telemetry.panel'
+                }).length > 0;
+            }
 
             function dragOver(e) {
+                //Refresh domain object on each dragOver to catch external
+                // updates to the model
+                //Don't use EditableDomainObject for folders, allow immediate persistence
+                editableDomainObject = domainObject.hasCapability('editor') || domainObject.getModel().type==='folder' ? domainObject : new EditableDomainObject(domainObject, $q);
+                actionCapability = editableDomainObject.getCapability('action');
```

Comment:
I think the gesture itself should be unchanged here; it's the consequence of the gesture that is different. Think there is a platform deficiency here insofar as gestures are responsible for their consequences; in this case, it's at least mitigated because the gesture fires an action which we can choose to swap out with something else.

Here, I think there are two approaches; maybe both are necessary, maybe they can be consolidated:

1. Decorate the existing `compose` gesture such that it also triggers a change into new edit mode (at least in the cases where it should)
2. Add an alternate `compose` action for cases where a telemetry panel needs to be created; give it a higher priority such that it is returned first, but filter it out in cases when it would not apply (either by policy, or via `appliesTo`). Later, when we want to generalize the behavior beyond just plots & panels, we have a specific place to hook into.